src/app/sign-in/components/form-sign-in/form-sign-in.component.ts

```diff
@@ -232,6 +234,14 @@ export class FormSignInComponent implements OnInit, AfterViewInit {
         map((value) => value.oauthSession)
       )
       .subscribe((requestInfoForm) => {
+        if (
+          requestInfoForm.error === 'invalid_grant' ||
+          requestInfoForm.error === 'invalid_scope'
+        ) {
+          this.isOauthError.next(true)
+          this.loading.next(false)
+          this.errorDescription.next(requestInfoForm.errorDescription)
+        }
```

Comment:
Two observations about this PR 

1- So we are now showing error for invalid_grant and invalid scope, but I believe Angel was requesting a redirect instead of showing the error on the card
2- I don't' think this PR is handling the invalid redirects URLs

Reply:
@DanielPalafox for now lets ignore 1, but, please check 2 and fix if needed.

Thanks